IFComp/lib/IFComp/Controller/Play.pm

```diff
@@ -36,6 +36,7 @@ sub fetch_entry :Chained('root') :PathPart('') :CaptureArgs(1) {
             || ( $c->stash->{ current_comp }->status eq 'processing_votes' )
             || ( $c->stash->{ current_comp }->status eq 'over' )
             || ( $c->user && ( $entry->author->id eq $c->user->id ) )
+            || ( $c->check_user_roles( 'curator' ) )
```

Comment:
Is this check method implicitly defined by catalyst based on the model definition? If so, does it need to be $c->user && $c->check_user_roles, or is it smart about the case where there's no user present?